components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ServiceProvider.java

```diff
@@ -327,6 +355,28 @@ public void setOutboundProvisioningConfig(OutboundProvisioningConfig outboundPro
         this.outboundProvisioningConfig = outboundProvisioningConfig;
     }
 
+    /**
+     * Returns the application role mapping configuration which indicates whether an IdP should
```

Comment:
```suggestion
     * Return the application role mapping configuration which indicates whether an IdP should
```

Reply:
Addressed in 4cdf9672bdd6a4d6060995024d14bd9dc79a1168